### ISSUE TITLE
Buggy Interaction with Gearifiers/Gear Core

### ISSUE THREAD

**Issue by fzzyhmstrs** [wisp-forest/owo-lib#107]
After a few days of wading through various modpacks with the help of several people in this issue: fzzyhmstrs/gearifiers#2 , it seems like we've narrowed down to **owo-lib** as a source of conflict with attribute modifiers.

The gist of the problem can be seen in various videos posted in that issue thread; the TL;DW is that attribute modifiers added by the affixes from Gearifers/Gear Core are applied when gear is equipped, but are not (always) un-applied when the gear is removed. This only happens with owo installed. Without it, affixes come on and off equally, as expected. 

I've tested this just with owo, fabric api, and Gearifiers + needed deps for that. Latest versions for all mods (1.19.2)
![image](https://user-images.githubusercontent.com/72876796/222874406-2a3f9dab-fab3-493e-9959-690f231e70bd.png)

This leads to attributes stacking up improperly until the player relogs. For example, you can see below that I am wearing zero armor, however, I still have 4 armor. This is after equipping and unequipping the leather pants right below the boot slot several times. They are affixed with my Legendary modifier (below):
![image](https://user-images.githubusercontent.com/72876796/222872478-142f540a-99b5-4554-a77a-a052e001df50.png)
![image](https://user-images.githubusercontent.com/72876796/222872508-712a777d-5f51-49cd-a7b4-276b5370868a.png)

This doesn't seem to happen when my affixes have any old type of attribute incorporated.. So far movement speed seems to be the culprit. If a piece of gear has an affix with speed included, the issue happens. There may be others, but I haven't tested through all my affixes yet.

**Comment by fzzyhmstrs** [comment]
Some additional insights gained after some more testing:
* Also happens with latest owo on 1.19.3 (0.10.2)
* Sometimes only happens when the item is damaged. I tested two helmets with my "Racing" affix (below), one on a fully repaired diamond helmet, one on one with a bit of damage. The damaged one caused the issue, repaired one no.
* I'll reach a point where adding and removing the helmet with racing does nothing at all. The speed seems capped out or something.
* Also managing to replicate now with an affix that just _removes_ armor. Snip below, the affix removes just 0.75 armor, but I'm obviously way below 7.25 armor if you look at the far bottom left.
* With a damaged item + an armor-adding affix I was able to "correct" this stacked negative armor modifier back to 0 (and beyond)

![image](https://user-images.githubusercontent.com/72876796/222876562-4be900b1-e6b5-4124-a154-8bd3805f0efc.png)

![image](https://user-images.githubusercontent.com/72876796/222876552-35d75dac-f36a-44fe-b642-d1b2c21a5fea.png)
